templates/default/flyingduck.hcl.erb

```diff
@@ -2,17 +2,6 @@ services = [
     {
         id = "flyingduck/metrics"
         name = "flyingduck"
-        tags = [
-            "monitoring"
-        ]
         port = <%= node['flyingduck']['port'] %>
```

Comment:
we might need a second port here for monitoring

Reply:
Yes but until we have a working endpoint to fetch the metrics from, we should remove the tag, otherwise prometheus is going to try to scrape it.